internal/guest/prot/protocol.go

```diff
@@ -722,8 +722,9 @@ type SchemaVersion struct {
 // Cmp compares s and v and returns:
 //
 // -1 if s <  v
-//  0 if s == v
-//  1 if s >  v
+//
+//	0 if s == v
+//	1 if s >  v
```

Comment:
Same as above, these three should be isolated together if they are to be rendered in monospace
```
//	-1 if s <  v
//	 0 if s == v
// 	 1 if s >  v
```